src/DictVectors/abstractdvec.jl

```diff
@@ -321,9 +212,6 @@ are present, eg. for generic `AbstractDVec`, this falls back to
 """
 kvpairs(v) = pairs(v)
```

Comment:
The original idea was to have `pairs(dv)` return raw data including initiator flags, etc., just like `pairs(storage(dv))` would,  and `kvpairs()` cleaned-up data. Now, I don't think that `kvpairs` is needed any more. 

I suggest to delete it together with its docstring.